ExampleService/ExampleService_WebApi/Controllers/TaskController.cs

```diff
@@ -0,0 +1,53 @@
+using Microsoft.AspNetCore.Mvc;
+
+namespace ExampleService_WebApi;
```

Comment:
```suggestion
namespace ExampleService.WebApi;
```

It is always a best practice to use `.` instead of `_` in namespace. We can use `.` in project name also instead of `_`

Reply:
Since the project was created in that name, I have used the same.